core/templates/dev/head/exploration_editor/main_tab/StateResponses.js

```diff
@@ -284,20 +286,26 @@ oppia.controller('StateResponses', [
       trainingDataService.initializeTrainingData(
         explorationId, currentStateName);
     };
+
     $scope.suppressDefaultAnswerGroupWarnings = function() {
       var interactionId = $scope.getCurrentInteractionId();
+      var answerGroups = responsesService.getAnswerGroups();
+      var handledAnswersArray = [];
```

Comment:
Consider adding a brief comment above each of these lines to explain what they represent? There's a lot of arrays related to 'choices' in some fashion, but they all seem different somehow...